swig/cpp/src/Session.h

```diff
@@ -50,11 +51,11 @@ class Session:public Iter_Value, public Values
     void list_schemas(Schema& schema);
     void get_schema(Schema& schema, const char *module_name, const char *revision,
```

Comment:
It would be better if the function returned a string. I think filling the Schema object is misleading here. Could you change it?